Dockerfile

```diff
@@ -41,43 +90,28 @@ RUN mkdir /opt/sambamba/ \
 #Samtools 1.3.1#
 ################
    ENV SAMTOOLS_INSTALL_DIR=/opt/samtools
-
-   WORKDIR /tmp
-   RUN wget https://github.com/samtools/samtools/releases/download/1.3.1/samtools-1.3.1.tar.bz2 && \
-       tar --bzip2 -xf samtools-1.3.1.tar.bz2
-
-   WORKDIR /tmp/samtools-1.3.1
-   RUN ./configure --with-htslib=$HTSLIB_INSTALL_DIR --prefix=$SAMTOOLS_INSTALL_DIR && \
+   RUN cd /tmp && wget https://github.com/samtools/samtools/releases/download/1.3.1/samtools-1.3.1.tar.bz2 && \
+       tar --bzip2 -xf samtools-1.3.1.tar.bz2 && cd /tmp/samtools-1.3.1 && \
+       ./configure --with-htslib=$HTSLIB_INSTALL_DIR --prefix=$SAMTOOLS_INSTALL_DIR && \
        make && \
-       make install
-
-   WORKDIR /
-   RUN rm -rf /tmp/samtools-1.3.1
-
-
-
-
+       make install && \
+       cd / && rm -rf /tmp/samtools-1.3.1 && ln -s /opt/samtools/bin/samtools /usr/bin/samtools
 
 ##########
 #Bedtools#
 ##########
 
-ARG PACKAGE_VERSION=2.27.1
-ARG BUILD_PACKAGES="git openssl python build-essential zlib1g-dev"
-ARG DEBIAN_FRONTEND=noninteractive
-RUN apt-get update && \
-    apt-get install --yes \
-              $BUILD_PACKAGES && \
-    cd /tmp && \
-    git clone https://github.com/arq5x/bedtools2.git && \
-    cd bedtools2 && \
-    git checkout v$PACKAGE_VERSION && \
-    make && \
-    mv bin/* /usr/local/bin && \
-    cd / && \
-    rm -rf /tmp/* && \
-    apt remove --purge --yes \
-              $BUILD_PACKAGES && \
-    apt autoremove --purge --yes && \
-    apt clean && \
-    rm -rf /var/lib/apt/lists/*
+RUN mkdir /opt/bedtools && cd /opt/bedtools && wget https://github.com/arq5x/bedtools2/releases/download/v2.29.2/bedtools.static.binary && \
+    chmod +x /opt/bedtools/bedtools.static.binary && \
+    ln -s /opt/bedtools/bedtools.static.binary /usr/bin/bedtools
+
+####################
+#Biscuit QC scripts#
+####################
+RUN cd /opt && \
+    git clone https://github.com/zwdzwd/biscuit.git
```

Comment:
Should this point to a particular release or commit?

Reply:
That's complicated, because the latest release doesn't have some updates. Furthermore, the repo has been forked and this one isn't under active development anymore.  In the next 6 months, we're going to have to grab the other version and update to the new QC tools, but that's a separate PR, so I'm going to leave this as-is for now. There's already an issue in to update these scripts at https://github.com/genome/analysis-workflows/issues/993